src/main/kotlin/com/example/pass_manager/service/ClientService.kt

```diff
@@ -0,0 +1,15 @@
+package com.example.pass_manager.service
+
+import com.example.pass_manager.domain.MongoClient
+import org.bson.types.ObjectId
+import java.math.BigDecimal
+import java.time.Instant
+
+interface ClientService {
+    fun findById(clientId: ObjectId): MongoClient?
+    fun create(newClient: MongoClient): MongoClient
+    fun update(clientId: ObjectId, modifiedClient: MongoClient): MongoClient
+    fun cancelPass(clientId: ObjectId, passId: ObjectId): Boolean
+    fun calculateSpentAfterDate(afterDate: Instant, clientId: ObjectId): BigDecimal
+    fun deleteById(clientId: ObjectId)
```

Comment:
too many responsibilities in the service

Reply:
So some methods should be transferred to other services?

Reply:
yep

Reply:
@ajax-gnatiuk-s is such service structure acceptable?
<img width="610" alt="image" src="https://github.com/user-attachments/assets/40f5ef4f-19e3-4919-9822-2cae54515260">